### PR TITLE
W2 - hieucap -  Create calculator

### PR DESCRIPTION
**Pull request by at-hieucap**
#### Related issues (Optional)
Task #14  

#### Description (Required)
- [x] Create calculator

### CODE REVIEW

**Comment by at-hieucap** [comment]
@at-huongnguyen  nhờ anh vào review giúp em với.
em cảm ơn anh nhiều.

File: app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_calculator);
+        initView();
+        addOnClickListener();
+    }
+
+    private void addOnClickListener() {
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java]: Should rename to: addListener

File: app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        mBtnSum.setOnClickListener(this);
+        mBtnSub.setOnClickListener(this);
+        mBtnMul.setOnClickListener(this);
+        mBtnDivision.setOnClickListener(this);
+        mEdtFirstTerm.addTextChangedListener(this);
+        mEdtSecondTerm.addTextChangedListener(this);
+    }
+
+    private void initView() {
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java]: initViews

File: app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        mEdtSecondTerm = findViewById(R.id.edtSecondTerm);
+        mTvOperation = findViewById(R.id.tvOperation);
+        mTvResult = findViewById(R.id.tvResult);
+    }
+
+    @Override
+    public void onClick(View v) {
+        if (checkAllTermIsEmpty(mEdtFirstTerm, mEdtSecondTerm)) {
+            Toast.makeText(getApplicationContext(), R.string.event_two_term_is_empty, Toast.LENGTH_SHORT).show();
+
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java]: Optimize this function

File: app/src/at-hieucap/res/layout/activity_calculator.xml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    </LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/lnOperation"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/lnFillOutNumber"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:layout_marginTop="15dp"
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/res/layout/activity_calculator.xml]: Remove layout_marginLeft, layout_marginRight

File: app/src/at-hieucap/res/layout/activity_calculator.xml [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+            style="@android:style/ButtonBar"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginLeft="10dp"
+            android:layout_weight="1"
+            android:text="@string/division"
+            android:textSize="20dp" />
+    </LinearLayout>
+
+    <ScrollView
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/res/layout/activity_calculator.xml]: Remove ViewGroup

File: app/src/at-hieucap/res/values/colors.xml [hunk 6]
```diff
@@ -8,4 +8,5 @@
     <color name="colorBlack">#020202</color>
     <color name="colorWhite">#ffff</color>
     <color name="colorBlueFb">#3B5998</color>
+    <color name="colorPink">#FF3DBE</color>
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/res/values/colors.xml]: Should rename to: pink/blue/white/black

File: app/src/at-hieucap/res/layout/activity_calculator.xml [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+            android:layout_weight="1"
+            android:background="@drawable/bg_edt_fill"
+            android:gravity="center"
+            android:inputType="numberDecimal"
+            android:textSize="20dp" />
+
+        <TextView
+            android:id="@+id/tvOperation"
+            android:layout_width="60dp"
+            android:layout_height="50dp"
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/res/layout/activity_calculator.xml]: use wrap_content

File: app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java [hunk 8]
```diff
@@ -0,0 +1,133 @@
+package vn.asiantech.internship.calculator;
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java]: Update to: /internship/calculator/ui/CalculatorActivity.java

File: app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+        return ((edtSecondTerm).getText().toString().equals(getApplicationContext().getText(R.string.value_0)));
+    }
+
+    @Override
+    public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+    }
+
+    @Override
+    public void onTextChanged(CharSequence s, int start, int before, int count) {
+        mTvResult.setText(R.string.text_null);
```
Comment (at-huongnguyen) [inline app/src/at-hieucap/java/vn/asiantech/internship/calculator/CalculatorActivity.java]: Please Check enable button

**Comment by at-hieucap** [comment]
@at-huongnguyen  please review for me.
Thank you so much!

**Pull request merged**